script.js

```diff
@@ -17,20 +20,62 @@ $(document).ready(function() {
   	var autorefresh_time = 1000000000; //crazy long == no refresh, because I'm lazy
   	var autorefresh_timeout;
 	var sort = ByCreationDate;
+	var previousFlags;
+	var previousFlagText = "";
+	
+	refreshPreviousFlags = function(callback) {
+		$.ajax({
+			type: "POST",
+			url: "http://jenkinsstuff.bl.ee/blaze/get_data.php",
```

Comment:
Looks like you missed a hardcoded url - just `/blaze/get_data.php` ought to work.